<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookTicketBox</class>
 <widget class="QDialog" name="BookTicketBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>220</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Departure Day</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="Date">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="minimumTime">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::DaySection</enum>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="GetList">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get List Flight</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="isSkyBoss">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sky Boss</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>284</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Start AirPort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="StartAirport">
      <item>
       <property name="text">
        <string>San bay Buon Ma Thuot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Ca Mau</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Cam Ranh</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Can Tho</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Cat Bi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Chu Lai</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Con Dao</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Da Nang</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Dien Bien Phu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Dong Hoi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Lien Khuong</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Noi Bai</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Phu Bai</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Phu Cat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay  Phu Quoc</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Pleiku</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Rach GIa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Tan Son Nhat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Tho Xuan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Tuy Hoa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Van Don</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Vinh</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="isEco">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eco</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>330</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Destination Airport</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="DesAirport">
      <item>
       <property name="text">
        <string>San bay Buon Ma Thuot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Ca Mau</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Cam Ranh</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Can Tho</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Cat Bi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Chu Lai</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Con Dao</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Da Nang</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Dien Bien Phu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Dong Hoi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Lien Khuong</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Noi Bai</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Phu Bai</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Phu Cat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay  Phu Quoc</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Pleiku</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Rach GIa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Tan Son Nhat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Tho Xuan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Tuy Hoa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Van Don</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San bay Vinh</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
